lib/firefox/webdriver/builder.js

```diff
@@ -218,11 +218,20 @@ module.exports.configureBuilder = function(builder, baseDir, options) {
       }
     } else {
       ffOptions.firefoxOptions_().androidPackage = 'org.mozilla.firefox';
```

Comment:
Are you leaving it as this (rather than org.mozilla.firefox_beta) in preparation for the release version?

Reply:
Yep, thinking there is only a couple of days more to wait?

Reply:
Sounds good. The last I heard was that it should make it into a release on around ~Sept 22nd which is about a week away.